include/posix/pthread.h

```diff
@@ -332,7 +332,9 @@ static inline int pthread_mutexattr_destroy(pthread_mutexattr_t *m)
 	struct pthread_barrier name = {				\
 		.wait_q = _WAIT_Q_INIT(&name.wait_q),		\
 		.max = count,					\
-	}
+}
+
+#define PTHREAD_BARRIER_SERIAL_THREAD 0
```

Comment:
Okay, according to doc
> The constant PTHREAD_BARRIER_SERIAL_THREAD is defined in <pthread.h> and its value shall be distinct from any other value returned by pthread_barrier_wait().

and

> the pthread_barrier_wait() function shall return PTHREAD_BARRIER_SERIAL_THREAD for a single (arbitrary) thread synchronized at the barrier and zero for each of the other threads.

So shouldn't the value be something other than 0? Else, we return zero to all the threads waiting on the barrier

Reply:
Uh... yeah.  This is what I get for pushing a feature without a test, I guess.  Will fix.  I also see I misspelled the name of the symbol in the commit message (though it looks correct here).